gateway/grpc.go

```diff
@@ -206,6 +206,11 @@ func (g *GrpcGateway) Ping() error {
 	return g.client.Ping(ctx)
 }
 
+func (g *GrpcGateway) WaitServer() error {
```

Comment:
do we want the context to come in as an arugment?

Reply:
I was just keeping it the same pattern as `Ping`. Do you think it's necessary?

Reply:
Best practise is to always send in ctx ad first arg when doing network calls

Reply:
yeah I think we need to change all to have context in general. not sure about how we version here, but this is a breaking change.

Reply:
Yeah if we want to add context to `Ping` we should do it on the stable cadence branch